pkg/exchange/bybit/exchange.go

```diff
@@ -277,3 +280,36 @@ func (e *Exchange) CancelOrders(ctx context.Context, orders ...types.Order) (err
 
 	return errs
 }
+
+func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, _, _ time.Time, lastOrderID uint64) (orders []types.Order, err error) {
+	log.Warn("!!!BYBIT EXCHANGE API NOTICE!!! the since/until conditions will not be effected on SPOT account, bybit exchange does not support time-range-based query currently")
+
+	if err := closedRateLimiter.Wait(ctx); err != nil {
+		return nil, fmt.Errorf("query closed order rate limiter wait error: %v", err)
```

Comment:
Go introduces an `%w` directive to wrap error https://stackoverflow.com/a/61287626/3897950